nodeEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -163,6 +170,40 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
         }, undefined, true);
     }
 
+    
+    loadFrame(file: File) {
+        Tools.ReadFile(file, (data) => {
+            // get Frame Data from file
+            let decoder = new TextDecoder("utf-8");
+            const frameData = JSON.parse(decoder.decode(data));

```

Comment:
Should not serialize deserialize but rather append to the current Node Material and editor data.